.github/workflows/typos.toml

```diff
@@ -0,0 +1,57 @@
+[default]
+extend-ignore-identifiers-re = [
+   "OTSs", # mentioned in CHANGELOG ... ???
+]
+
+extend-ignore-re = [
+  '"shape": ?"[^"]+"',
+  '"encoded_polyline": ?"[^"]+"',
+  '"id": ?"[^"]+"',
+  'R"\([^)]+\)"',
+]
+
+[default.extend-words]
```

Comment:
could you pls add a one-line explanation each to disambiguate
- extend-ignore
- extend-words
- extend-identifiers

as a comment in this file?

Reply:
I added a link to the config file documentation at the top of the file ... is that good enough?

Reply:
Jep, sorry, didn’t see that

Reply:
I added it only after your feedback (and force pushed the updated commit) ... you didn't miss anything ;)